x/globalfee/types/params.go

```diff
@@ -2,18 +2,45 @@ package types
 
 import (
 	"fmt"
+	"strings"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	ibcclienttypes "github.com/cosmos/ibc-go/v4/modules/core/02-client/types"
+	ibcchanneltypes "github.com/cosmos/ibc-go/v4/modules/core/04-channel/types"
 )
 
-// ParamStoreKeyMinGasPrices store key
-var ParamStoreKeyMinGasPrices = []byte("MinimumGasPricesParam")
+var (
+	// ParamStoreKeyMinGasPrices store key
+	ParamStoreKeyMinGasPrices                    = []byte("MinimumGasPricesParam")
+	ParamStoreKeyBypassMinFeeMsgTypes            = []byte("BypassMinFeeMsgTypes")
+	ParamStoreKeyMaxTotalBypassMinFeeMsgGasUsage = []byte("MaxTotalBypassMinFeeMsgGasUsage")
+
+	DefaultMinGasPrices         = sdk.DecCoins{}
```

Comment:
Could you add a comment that this means that the staking token is used as default denom with 0 as amount, i.e., `0uatom`?